### PR TITLE
DR-2467: Add script for getting RBS Pool info per environment

### PR DESCRIPTION
**Pull request by snf2ye**
In order to get information about the number of projects available for handout, we must authenticate as the right RBS SA on the RBS swagger page. This script should provide all of the needed info to auth: The swagger url per environment, the bearer token for the SA, and the pool id. In order to run the script, you must be logged into vault.

### CODE REVIEW

**Review by nmalfroy — APPROVED** [review]
Awesome, thanks for this!  Is there a Jira to tie this to?

File: tools/rbs-bearer-token.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Use this script to get the RBS Bearer token to use to auth w/ RBS swagger page
+# Swagger page allows you to check how many projects are available for the pool
```
Comment (fboulnois) [inline tools/rbs-bearer-token.sh:5]:
```suggestion
#!/bin/sh
set -eu

# Use this script to get the RBS Bearer token to use to auth w/ RBS swagger page
# Swagger page allows you to check how many projects are available for the pool
```

Comment (fboulnois) [reply tools/rbs-bearer-token.sh:5]: I also recommend running `shellcheck` on your script.

File: tools/rbs-bearer-token.sh [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  prod)
+    BUFFER_CLIENT_SERVICE_ACCOUNT_VAULT_PATH="secret/suitable/terra/kernel/prod/prod/buffer/client-sa"
+    RBS_SWAGGER_URL="https://buffer.dsde-prod.broadinstitute.org/swagger-ui.html"
+    RBS_POOL_ID="datarepo_v1"
+    ;;
+  esac
+  echo "Will get secret from $BUFFER_CLIENT_SERVICE_ACCOUNT_VAULT_PATH"
+  break
+done
+echo
```
Comment (fboulnois) [inline tools/rbs-bearer-token.sh]:
```suggestion
printf "\n"
```

Perhaps replace empty `echo`s with `printf` for clarity? (I'm not sure if empty `echo`s are POSIX compatible)

**Review by fboulnois — COMMENTED** [review]
A few suggestions:

**Comment by snf2ye** [comment]
> Awesome, thanks for this! Is there a Jira to tie this to?

Ticket added!

File: tools/rbs-bearer-token.sh (hunk 1 above)
Comment (snf2ye) [reply tools/rbs-bearer-token.sh:5]: Ok - I ran shellcheck. Everything passes except for a warning message about the select loop `SC3008 (warning): In POSIX sh, select loops are undefined.` Given it's just a warning, I would like to keep the select loop.

Comment (fboulnois) [reply tools/rbs-bearer-token.sh:5]: If there's no need for this script to be POSIX compatible, you can just switch to `#!/bin/bash` and then `shellcheck` should stop complaining.

**Review by fboulnois — COMMENTED** [review]
Short comment to potentially suppress the warning:

File: tools/rbs-bearer-token.sh (hunk 1 above)
Comment (snf2ye) [reply tools/rbs-bearer-token.sh:5]: That worked!

**Review by fboulnois — APPROVED** [review]

**Review by tlangs — APPROVED** [review]

**Pull request merged**